basalt.lua

```diff
@@ -0,0 +1,18 @@
+-- https://github.com/Pyroxenium/Basalt/
+
+-- TODO: [Github releases provider](https://github.com/unicornpkg/wing/issues/7)
+-- https://github.com/Pyroxenium/Basalt/releases/download/v1.6.3/basalt.lua
+
+local package = {}
+package.name = "basalt"
+package.version = "1.6.3"
+package.instdat = {}
+package.instdat.repo_owner = "pyroxenium"
+package.instdat.repo_name = "basalt"
+package.instdat.repo_ref = "1.6.3"
```

Comment:
This repository ref is inaccurate. It should point to a valid Git tag.
```suggestion
package.instdat.repo_ref = "v1.6.3"
```